Harmony/Extras/FastAccess.cs

```diff
@@ -154,14 +162,12 @@ public static InstantiationHandler<T> CreateInstantiationHandler<T>()
 			return (SetterHandler<T, S>)dynamicSet.Generate().CreateDelegate(typeof(SetterHandler<T, S>));
 		}
 
-		//
-
-		static DynamicMethodDefinition CreateGetDynamicMethod<T, S>(Type type)
+		private static DynamicMethodDefinition CreateGetDynamicMethod<T, S>(Type type)
```

Comment:
I think the Harmony code standard does not use `private` and instead let no modifier be implicit private.

Reply:
Fixed in all changed files, except in the asset classes, where I think the explicit `private` makes sense (some other asset classes also use explicit `private`).